07week/dodgeBall/dodgeBall.js

```diff
@@ -0,0 +1,183 @@
+'use strict'
+
+
+const arrOfPeople = [
+  {
+      id: 2,
+      name: "Charles Young",
+      age: 55,
+      skillSet: "welding",
+      placeBorn: "Omaha, Nebraska"
+  },
+  {
+      id: 3,
+      name: "Judy Twilight",
+      age: 35,
+      skillSet: "fishing",
+      placeBorn: "Louisville, Kentucky"
+  },
+  {
+      id: 4,
+      name: "Cynthia Doolittle",
+      age: 20,
+      skillSet: "tic tac toe",
+      placeBorn: "Pawnee, Texas"
+  },
+  {
+      id: 5,
+      name: "John Willouby",
+      age: 28,
+      skillSet: "pipe fitting",
+      placeBorn: "New York, New York"
+  },
+  {
+      id: 6,
+      name: "Stan Honest",
+      age: 20,
+      skillSet: "boom-a-rang throwing",
+      placeBorn: "Perth, Australia"
+  },
+  {
+      id: 7,
+      name: "Mia Watu",
+      age: 17,
+      skillSet: "acrobatics",
+      placeBorn: "Los Angeles, California"
+  },
+  {
+      id: 8,
+      name: "Walter Cole",
+      age: 32,
+      skillSet: "jump rope",
+      placeBorn: "New Orleans, Louisiana"
+  },
+]
+
+const listOfPlayers = [];
+const blueTeam = [];
+const redTeam = [];
+
+
+
+class player {
+  constructor(id, name, age, skillSet, placeBorn, canThrowBall, canDodgeBall, hasPaid, isHealthy, yearsExperience) {
+      this.id = id;
+      this.name = name;
+      this.age = age;
+      this.skillSet = skillSet;
+      this.placeBorn = placeBorn;
+      this.canThrowBall = canThrowBall;
+      this.canDodgeBall = canDodgeBall;
+      this.hasPaid = hasPaid;
+      this.isHealthy = isHealthy;
+      this.yearsExperience = yearsExperience;
+  }
+}
+class blueTeammate extends player {
```

Comment:
There really should only be a single TeamMate class that has an attribute of color, instead of separate classes for red team and blue team.